csharp-basics/exercises/Arithmetic/CozaLozaWoza/Program.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+
+namespace CozaLozaWoza
+{
+    internal class Program
+    {
+        public static void Main(string[] args)
+        {
+            for (var i = 1; i <= 110; i++)
+            {
+                var word = false;
+                if (i % 3 == 0)
+                {
+                    Console.Write("Coza");
+                    word = true;
+                }
+                if (i % 5 == 0)
+                {
+                    Console.Write("Loza");
+                    word = true;
+                }
+                if (i % 7 == 0)
```

Comment:
else if

Reply:
"The program shall print "Coza" in place of the numbers which are multiples of 3, "Loza" for multiples of 5, "Woza" for multiples of 7, "CozaLoza" for multiples of 3 and 5, and so on."

"and so on." es sapratu kā CozaLozaWoza 3, 5 un 7 gadījumā.

Kā arī, google pameklējot cozalozawoza, var atrast, ka tā jābūt.

Reply:
Jā, viss ok te